src/NServiceBus.NHibernate.Tests/Persistence/NHibernateConfigurationBuilderTests.cs

```diff
@@ -130,120 +125,5 @@ public void Should_skip_settings_that_are_not_for_persistence()
 
             CollectionAssert.IsSubsetOf(expected, builder.Build().Configuration.Properties);
         }
-
-#if NETFRAMEWORK
```

Comment:
Since these tests are doing things with custom AppDomains, I understand why you removed them, but I'm wondering these are valid scenarios we still need to be testing for in some other way instead.

I don't know enough of the specific details around NHibernate, but I assume it still uses these config files when running on .NET.

If that's true, then maybe we need to do something more here?

Reply:
Also, the Tests csproj is currently copying `hibernate.cfg.xml`and `Testing.config` to the output directory. I see `Testing.config` referenced in these tests. Are both of these files just for these removed tests? 

If so, we should clean those up also.

Reply:
I have removed the unused config files since there are tests already that checks that we can override

https://github.com/Particular/NServiceBus.NHibernate/blob/master/src/NServiceBus.NHibernate.Tests/Persistence/NHibernateConfigurationBuilderTests.cs#L65

and this just tests that we can override what NH read from its config file => I don't see enough value to justify us fiddling with customer exes that checks this